test/test_extended_broadcast_view.cpp

```diff
@@ -6,8 +6,9 @@
 *                                                                          *
 * The full license is in the file LICENSE, distributed with this software. *
 ****************************************************************************/
+// This file is generated from the source file test/files/cppy_source/test_extended_broadcast_view.cppy by preprocess.py!
+// Warning: This file should not be modified directly! Please modify the source file to change the test!
```

Comment:
I'd prefer not to exceed 100 characters per line. So maybe:
```suggestion
// This file is generated from test/files/cppy_source/test_extended_broadcast_view.cppy by preprocess.py!
// Warning: This file should not be modified directly! Instead, modify the `*.cppy` file.
```
Same for all occurrences.